ocean-components/src/main/res/layout/ocean_basic_chip_item.xml

```diff
@@ -14,9 +14,27 @@
     </data>
 
     <LinearLayout
-        android:id="@+id/chip_list_item_background"
         android:layout_width="wrap_content"
         android:layout_height="32dp"
+        android:orientation="horizontal">
+
+        <LinearLayout
+            android:layout_width="2dp"
+            android:layout_height="match_parent"
+            android:layout_marginStart="@dimen/ocean_spacing_stack_xxs"
+            android:layout_marginEnd="@dimen/ocean_spacing_stack_xs"
+            app:ocean_visible_or_gone="@{item.hasFilterAll == true}">
```

Comment:
um Boolean de databinding nao precisa fazer a checagem explicita. A variável tem valor false por default se ela não for passada de parâmetro.
```suggestion
            app:ocean_visible_or_gone="@{item.hasFilterAll}">
```